components/provisioner/internal/provisioning/graphql_converter_test.go

```diff
@@ -422,3 +428,12 @@ func fixKymaRelease() model.Release {
 		InstallerYAML: "installer yaml",
 	}
 }
+
+func fixKymaReleaseWithoutTiller() model.Release {
+	return model.Release{
+		Id:            "e829b1b5-2e82-426d-91b0-f94978c0c140",
+		Version:       kymaVersionWithoutTiller,
+		TillerYAML:    "",
+		InstallerYAML: "installer yaml",
+	}
+}
```

Comment:
I think this function is used only in `input_converter_tests.go` so that is a probably better place for it.